### DISCUSSION TITLE
Ignore certain dates like birthdays

### DISCUSSION THREAD

**Discussion by henningBunk** [jonaswinkler/paperless-ng#786] [Archived FRs]
I recently added ~600 documents and more than 50 had my birthday assigned as creation date. This is for documents which contain my birthday but also a "correct" date. I think this will also happen with new documents in the future.

When I could set my birthday on an ignore list paperless should be able to find a correct creation date more often.

Ideally, new users would be encouraged to enter the birthday before uploading their initial batch of documents.

**Comment by jonaswinkler** [comment]
Search https://paperless-ng.readthedocs.io/en/latest/configuration.html for PAPERLESS_IGNORE_DATES :)

**Reply by henningBunk** [reply]
🤦  Thank you :)
